apps/daffio/src/app/docs/design/components/component-content/component-content.component.html

```diff
@@ -0,0 +1,32 @@
+<daffio-doc-article [breadcrumbs]="doc().breadcrumbs" [toc]="toc()">
+	<h1>{{doc().title}}</h1>
+	<div [outerHTML]="doc().longDescription | safe"></div>
```

Comment:
This content doesn't change when I navigate between pages using the sidebar. It only updates on refresh, or I type in the specific URL.

Reply:
yes, its a bug of using `outerHTML`...I can't find a good workaround, I will have to remove the `<p>` tag from the rendered `longDescription` and use `innerHtml` on a `<p>`

Reply:
fixed